library/Makefile

```diff
@@ -88,10 +88,11 @@ OBJS_CRYPTO=	aes.o		aesni.o		arc4.o		\
 		pk.o		pk_wrap.o	pkcs12.o	\
 		pkcs5.o		pkparse.o	pkwrite.o	\
 		platform.o	platform_util.o	poly1305.o	\
-		ripemd160.o	rsa_internal.o	rsa.o  		\
-		sha1.o		sha256.o	sha512.o	\
-		threading.o	timing.o	version.o	\
-		version_features.o		xtea.o
+                ripemd160.o	rsa_internal.o	\
```

Comment:
Why did you change the formatting here?

Reply:
I initially added `psa_util.o` to the crypto object file list, which was wrong, and when I moved it to the TLS object file list, the reformatting of the crypto list remained. I'll revert it.